Readme example 6.1 behind new API

Using image v0.19.0 and wanted to build example 6.1. cargo build gives the error below


```
error[E0061]: this function takes 1 parameter but 2 parameters were supplied
  --> src/main.rs:21:9
   |
21 |     img.save(fout, image::PNG).unwrap();
   |         ^^^^ expected 1 parameter

```

According to [current src of save function](https://github.com/PistonDevelopers/image/blob/master/src/dynimage.rs#L455-L466), the format is inferred from the name of the file

--- a/README.md
+++ b/README.md
@@ -183,8 +183,6 @@ The image format is determined from the path's file extension.
 ```rust
 extern crate image;
 
-use std::fs::File;
-
 use image::GenericImage;
 
 fn main() {
@@ -198,10 +196,8 @@ fn main() {
     // The color method returns the image's `ColorType`.
     println!("{:?}", img.color());
 
-    let ref mut fout = File::create("test.png").unwrap();
-
     // Write the contents of this image to the Writer in PNG format.
-    img.save(fout, image::PNG).unwrap();
+    img.save("test.png").unwrap();
 }
 ```
 

